aether-producer/producer/aether_producer.py

```diff
@@ -165,21 +164,21 @@ def safe_sleep(self, dur):
 
     # Connectivity
 
+    # see if kafka's port is available
     def kafka_available(self):
-        kafka_url = "http://%s" % self.settings["kafka_url"]
+        kafka_ip, kafka_port = self.settings["kafka_url"].split(":")
+        kafka_port = int(kafka_port)
```

Comment:
Default to `80`/`443` in the (admittedly unlikely) event that there's no port?

Reply:
The Kafka port defaults to 9092. It shouldn't ever be on the default http/s ports

Reply:
OK, that's I figured.